projects/opentrain/gtfs_pipeline/heatmap/HeatMap.java

```diff
@@ -0,0 +1,895 @@
+package projects.opentrain.gtfs_pipeline.heatmap;
+
+import com.projects.opentrain.gtfs_pipeline.Protos.Stop;
+import com.projects.opentrain.gtfs_pipeline.Protos.StopTime;
+import com.projects.opentrain.gtfs_pipeline.Protos.Trip;
+
+import com.google.startupos.common.flags.Flags;
+import com.google.startupos.common.flags.Flag;
+import com.google.startupos.common.flags.FlagDesc;
+
+import javax.imageio.ImageIO;
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.Font;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class HeatMap {
+
+  private List<StopsAround> stopsAroundList = new ArrayList<>();
+
+  private double[][] heatMap;
+  private List<Stop> stopsList = new ArrayList<>();
+  private List<StopTime> stopTimeList = new ArrayList<>();
+  private List<Trip> tripsList = new ArrayList<>();
+  private static final String tripsPath = "projects/opentrain/gtfs_pipeline/trips.protobin";
+  private static final String stopsPath = "projects/opentrain/gtfs_pipeline/stops.protobin";
+  private static final String stopTimePath = "projects/opentrain/gtfs_pipeline/stopTime.protobin";
+
+  @FlagDesc(
+      name = "image_path",
+      description = "Image path to map image,needs to contain .../<FileName>.png")
+  public static Flag<String> imagePath = Flag.create("");
+
+  @FlagDesc(
+      name = "heat_map_path",
+      description =
+          "Image destination path to create heatmap image,needs to contain .../<FileName>.png")
+  public static Flag<String> heatMapPath = Flag.create("");
+
+  public HeatMap(int stopId, String time) throws IOException { // Time = HH:MM:ss
+
+    dataToList();
+
+    // From all the stops
+    for (StopTime stopTime : stopTimeList) {
+      // Pick the stop that i want, That start some trip, At the chosen time
```

Comment:
Let's rephrase to:
`Pick the stop that we want, that starts at some trip, at the given time`